lib/workers/repository/update/pr/code-owners.ts

```diff
@@ -5,6 +5,24 @@ import { readLocalFile } from '../../../../util/fs';
 import { getBranchFiles } from '../../../../util/git';
 import { newlineRegex, regEx } from '../../../../util/regex';
 
+interface FileOwnerRule {
+  usernames: string[];
+  pattern: string;
+  score: number;
+  match: (path: string) => boolean;
+}
+
+const extractOwnersFromLine = (line: string): FileOwnerRule => {
```

Comment:
convert to full function declaration

Reply:
you should probably disable arrow functions with some eslint rule